src/boot/efi/boot.c

```diff
@@ -513,22 +519,27 @@ static void print_status(Config *config, CHAR16 *loaded_image_path) {
         case TIMEOUT_UNSET:
             break;
         case TIMEOUT_MENU_FORCE:
-            Print(L"     timeout (EFI var): menu-force\n"); break;
+            Print(L"       timeout (EFI var): menu-force\n"); break;
         case TIMEOUT_MENU_HIDDEN:
-            Print(L"     timeout (EFI var): menu-hidden\n"); break;
+            Print(L"       timeout (EFI var): menu-hidden\n"); break;
         default:
-            Print(L"     timeout (EFI var): %lu s\n", config->timeout_sec_efivar);
+            Print(L"       timeout (EFI var): %lu s\n", config->timeout_sec_efivar);
         }
 
-        ps_string(L"               default: %s\n", config->entry_default_config);
-        ps_string(L"     default (EFI var): %s\n", config->entry_default_efivar);
-        ps_string(L"    default (one-shot): %s\n", config->entry_oneshot);
-        ps_string(L"           saved entry: %s\n", config->entry_saved);
-          ps_bool(L"                editor: %s\n", config->editor);
-          ps_bool(L"          auto-entries: %s\n", config->auto_entries);
-          ps_bool(L"         auto-firmware: %s\n", config->auto_firmware);
-          ps_bool(L"                  beep: %s\n", config->beep);
-          ps_bool(L"  reboot-for-bitlocker: %s\n", config->reboot_for_bitlocker);
+        ps_string(L"                 default: %s\n", config->entry_default_config);
+        ps_string(L"       default (EFI var): %s\n", config->entry_default_efivar);
+        ps_string(L"      default (one-shot): %s\n", config->entry_oneshot);
+        ps_string(L"             saved entry: %s\n", config->entry_saved);
+          ps_bool(L"                  editor: %s\n", config->editor);
+          ps_bool(L"            auto-entries: %s\n", config->auto_entries);
+          ps_bool(L"           auto-firmware: %s\n", config->auto_firmware);
+          ps_bool(L"                    beep: %s\n", config->beep);
+          ps_bool(L"    reboot-for-bitlocker: %s\n", config->reboot_for_bitlocker);
+          ps_bool(L"        auto-enroll-keys: %s\n", config->auto_enroll_keys);
+        Print(L"*** DANGER ***\n");
+        Print(L"Setting this option to true might soft-brick your machine.\n");
```

Comment:
i don't think this is he right place to warn like that... 

i mean, warning when you actually enroll, and warning in the docs where you look when setting it, makes a ton of sense. But here it is already too late...